### ISSUE TITLE
Support multiple ownCloud installations

### ISSUE THREAD

**Issue by cweiske** [guillaumev/owncloud_for_filelink#43]
I have my personal ownCloud instance, and one at work.

I'd like to add both instances to Thunderbird - but after adding one, I cannot add another ownCloud instance.

**Comment by clmcavaney** [comment]
Agree, I have the same circumstance - this would be a nice feature.
